.github/workflows/cherry-picks.yml

```diff
@@ -0,0 +1,37 @@
+name: Cherry Pick to remote repository
+on:
+  pull_request_target:
+    types: [closed]
+permissions:
+  contents: write # so it can comment
+  pull-requests: write # so it can create pull requests
+jobs:
+  cross-repo-cherrypick:
+    name: Cherry pick to remote repository
+    runs-on: ubuntu-latest
+    # Don't run on closed unmerged pull requests
+    if: github.event.pull_request.merged
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.FINEGRAINED_PAT }}
+      - name: Create backport pull requests
+        uses: jschmid1/cross-repo-cherrypick-action@807f3f450c82fe6dd40220b1cee0449a3fb26b44
```

Comment:
I would prefer the contents of an action we are using was owned by the Kong org. There's https://github.com/Kong/public-shared-actions if you need a place to land it.

Reply:
the repo you linked appears just a place to store `action definitions` rather than the actual `action implementation`.  I'll be fine with moving this to a Kong owned repository after we've removed the initial hurdles. For now, I feel it's easier to have it point to my own repository for quicker turn-around times.

Reply:
Alright.